source/create_project/index.html.md.erb

```diff
@@ -1,10 +1,17 @@
 ---
-title: "Create a documentation project"
+title: Create a documentation project
 weight: 20
 last_reviewed_on: 2019-10-03
 review_in: 1 day
 ---
 
 # Create a documentation project
 
-You can create a single page or multipage documentation project.
+To create a documentation project:
+
+- [set up your local machine][before-you-start] to get started
+- [create a new documentation project][create-project]
+- [preview your documentation][preview-your-docs]
+- [build your documentation][build-your-docs]
```

Comment:
As discussed at crit - I wonder if 'build' is technically one of these steps (especially as build happens during the preview). Maybe it could be a separate line underneath - something like: "You can also build your documentation separately".